### PR TITLE
Update auto-update config of cellx.js

### PR DESCRIPTION
**Pull request by finch185277**
Related issue(s): #11932

The current version on CDNJS is outdated because the file structure on the upstream repo changed. I made this change so that auto-update can successfully grab the latest version.

### CODE REVIEW

File: ajax/libs/cellx/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     {
       "basePath": "dist",
       "files": [
-        "**/*"
+        "cellx*"
```
Comment (extend1994) [inline ajax/libs/cellx/package.json]: Please use exclusive pattern to exclusive possiblely undesirable files.

**Comment by extend1994** [comment]
ping @finch185277

File: ajax/libs/cellx/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "cellx",
-  "filename": "cellx.min.js",
-  "version": "1.7.57",
+  "filename": "cellx.umd.js",
+  "version": "1.7.82",
```
Comment (MattIPv4) [inline ajax/libs/cellx/package.json]:
```suggestion
  "version": "1.7.57",
```

Please revert the change to `version`, the auto-update bot will handle chaging this when it pulls in new versioned files.

**Comment by MattIPv4** [comment]
Ah, looks like this has since been updated in another PR -- will close.
